### PR TITLE
Method names and field names should start with a lower case letter

### PR DESCRIPTION
**Pull request by vbauer**
Just for additional information: https://docs.oracle.com/javase/tutorial/java/javaOO/methods.html

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vbauer__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java [hunk 1]
```diff
@@ -80,7 +80,7 @@
    * Test switch to log all bytes that come across the wire.
    * Helpful when parsing fails before certain bytes hit the trace logs.
    */
-  public static boolean LogAllWireBytes = false;
+  public static boolean logAllWireBytes = false;
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: maybe this one should be `final` and written `LOG_ALL_WIRE_BYTES` like introduced in [docs.oracle.com](http://docs.oracle.com/javase/specs/jls/se6/html/names.html#6.8.6)

Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: maybe we should make this private and implement a setter?

File: src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java [hunk 2]
```diff
@@ -837,7 +837,7 @@ public ByteArrayArray createComplexProperty() {
    * Defines MSExchangeCertificate Property **
    */
 
-  public static PropertyDefinition MSExchangeCertificate =
+  public static final PropertyDefinition MSExchangeCertificate =
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java]: as by convention `static final` classes should be written in 
> a sequence of one or more words, acronyms, or abbreviations, all uppercase, with components separated by underscore "_" characters
> http://docs.oracle.com/javase/specs/jls/se6/html/names.html#6.8.6

**Comment by serious6** [comment]
:+1:

File: src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java (hunk 2 above)
Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java]: Completely agree! I just didn't want to touch a lot of sources. I can fix it in this PR or in separate PR. What do you prefer?

File: src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java (hunk 1 above)
Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: I also thought about it, but this field is used outside this class. I can add getter and setter for this field or keep it as now. What do you prefer?

Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: :+1: for getter/setter

File: src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java (hunk 2 above)
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java]: don't have any prefs so far. There are pros/cons for both of it. ;-)

Comment (vboctor) [inline src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java]: I would go with a separate PR dedicated to fixing static finals.

**Comment by vboctor** [comment]
:+1:

File: src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java (hunk 2 above)
Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/core/service/schema/ContactSchema.java]: NP, I'll make a separate PR.

**Comment by vbauer** [comment]
PR was updated.

File: src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java [hunk 3]
```diff
@@ -122,6 +112,14 @@ void hangingRequestDisconnectHandler(Object sender,
   }
 
 
+  public static boolean isLogAllWireBytes() {
+    return logAllWireBytes;
+  }
+
```
Comment (serious6) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: maybe we should also keep track on parallelity. Calling the getter and setter will not be thread-safe so far.

Comment (vbauer) [inline src/main/java/microsoft/exchange/webservices/data/core/request/HangingServiceRequestBase.java]: You're right, fixed.

**Comment by vbauer** [comment]
Everything is updated.

**Comment by serious6** [comment]
thanks. Will be merged within the next days. Marked this for 2.0.

**Comment by serious6** [comment]
@vbauer plz update your pr.

**Comment by vbauer** [comment]
@serious6 Updated.

**Comment by serious6** [comment]
@vbauer thanks for your contribution.

**Comment by vbauer** [comment]
@serious6 You're welcome.
